### ISSUE TITLE
[BUG] - tiktok changed api

### ISSUE THREAD

**Issue by 11348** [davidteather/TikTok-Api#520]
api_url add "/post"
```api_url = "{}api/post/item_list/?{}&{}"```

"items" change to "itemList"
```
if "itemList" in res.keys():
    for t in res["itemList"]:
        response.append(t)
```

"maxCursor" change to "cursor"
```maxCursor = res["cursor"]```

**Comment by hinchyy** [comment]
Yeah i am also running into the same issue. Tried your bug fixes still getting a lot of errors.

**Comment by davidteather** [comment]
Should be fixed with [V3.9.2](https://github.com/davidteather/TikTok-Api/releases/tag/V3.9.2)
